gc/base/MarkedObjectPopulator.cpp

```diff
@@ -72,9 +72,9 @@ MM_MarkedObjectPopulator::populateObjectHeapBufferedIteratorCache(omrobjectptr_t
 		} else {
 			/* set startPtr to just past the last object seen */
 #if defined(OMR_GC_MINIMUM_OBJECT_SIZE)
-			state->data1 = ((uintptr_t)object) + J9_GC_MINIMUM_OBJECT_SIZE;
+			state->data1 = ((uintptr_t)object) + OMR_MINIMUM_OBJECT_SIZE;
 #else /* OMR_GC_MINIMUM_OBJECT_SIZE */
-			state->data1 = ((uintptr_t)object) + sizeof(J9Object);
+			state->data1 = ((uintptr_t)object) + sizeof(fomrobject_t);
```

Comment:
I have to look into this more closely. This change will have no effect on existing compiled code, since J9Object and fomrobject_t definitions are identical. I see your point (result should have object (8-byte) alignment, and I suspect that this is taken care of in the MM_ObjjectHeapBufferedIterator and its many Populator pals. Will verify this (or not).